backend/tracim_backend/config.py

```diff
@@ -307,6 +307,12 @@ def __init__(self, settings):
         self.EMAIL_NOTIFICATION_ACTIVATED = asbool(settings.get(
             'email.notification.activated',
         ))
+        if not self.EMAIL_NOTIFICATION_ACTIVATED:
+            logger.warning(
+                self,
+                'Notification by email mecanism is disabled !'
```

Comment:
Missing space at end of string. This will result in no separator space between sentences